datafusion/core/tests/sql/mod.rs

```diff
@@ -634,7 +634,7 @@ pub fn make_partition(sz: i32) -> RecordBatch {
 
 /// Specialised String representation
 fn col_str(column: &ArrayRef, row_index: usize) -> String {
-    if column.is_null(row_index) {
+    if column.data_type() == &DataType::Null || column.is_null(row_index) {
```

Comment:
The `abs` function may now return a [NullArray](https://github.com/apache/arrow-rs/blob/7355e8392d6e82c28cc54236d79b4dc56d7226d4/arrow-array/src/array/null_array.rs#L45). For example: `SELECT abs(NULL)`.
And `is_null()` does not work on NullArray.

Reply:
Filed https://github.com/apache/arrow-rs/issues/4835